### PR TITLE
Ihc configflow

### PR DESCRIPTION
**Pull request by dingusdk**
## Proposed change

This is the IHC integration documentation for config flow.
Manual configuration has been moves out of the configuration.yaml file.
Service functions also has changed because controllers are not identified by serial.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

The changes in this documentation is impemented in this code PR:

https://github.com/home-assistant/core/pull/50957

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/ihc.markdown [hunk 1]
```diff
@@ -222,15 +211,15 @@ When auto setup is enabled the following products will be found in the IHC proje
 - Wireless combi relay 4 buttons
 - Wireless mobile dimmer
 - Dataline lamp outlet
+- RS486 dual channel LED dimmer
```
Comment (Abildgren) [inline source/_integrations/ihc.markdown]:
```suggestion
- RS485 dual channel LED dimmer
```

File: source/_integrations/shelly.markdown [hunk 2]
```diff
@@ -70,9 +70,44 @@ Shelly device relays are added to the Home Assistant by default as `switch` enti
 
 ## Events
 
-If the **BUTTON TYPE** of the switch connected to the device is set to `momentary` or `detached switch`, integration fires events when the switch is used. You can use these events in your automations.
+If the **BUTTON TYPE** of the switch connected to the device is set to `momentary` or `detached switch`, integration fires events under the type `shelly.click` when the switch is used. You can use these events in your automations.
```
Comment (Abildgren) [inline source/_integrations/shelly.markdown]: This file should not be part of the commit

File: source/_posts/2021-05-21-community-highlights.markdown [hunk 3]
```diff
@@ -0,0 +1,102 @@
+---
```
Comment (Abildgren) [inline source/_posts/2021-05-21-community-highlights.markdown]: this file should not be part of the commit

File: source/_integrations/default_config.markdown [hunk 4]
```diff
@@ -32,6 +32,7 @@ This integration is a meta-component and configures a default set of integration
 - [Scene](/integrations/scene/) (`scene`)
 - [Scripts](/integrations/script/) (`script`)
 - [Simple Service Discovery Protocol (SSDP)](/integrations/ssdp/) (`ssdp`)
+- [Stream](/integrations/stream/) (`stream`)
```
Comment (Abildgren) [inline source/_integrations/default_config.markdown]: I failed to see the dependency. Is this a mistake or have I overlooked something?

**Review by Abildgren — COMMENTED** [review]
A few files should be omitted from this commit. Else LGTM

**Comment by homeassistant** [comment]
Hi @235816, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#17924).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/default_config.markdown (hunk 4 above)
Comment (dingusdk) [reply source/_integrations/default_config.markdown]: Only the ihc.markdown file should be part of this PR. I only changed that file so i don' t understand why there are other files here. I tried updating my "ihcconfigflow" branch from current, but that did not help. Can i just select "delete file" here in the PR?

**Comment by dingusdk** [comment]
I think something is messed up in my repository - because now there are more files i did not change, and a user @235816 that needs to sign CLA.

I think i will be easier to close this PR and make a new one. After i have reverted my repository and added the new ihc.markdown file again.

**Comment by dingusdk** [comment]
I have made a new PR to replace this #18310

I think the problem was that my ihcconfig branch was branched from "current" and i make this pr to "next".

This ne PR only contains the ihc.markdown file
